static/src/stylesheets/module/atoms/_quiz.scss

```diff
@@ -0,0 +1,157 @@
+
+$accessibility-outline: #ffbf47;
```

Comment:
Some of them are based on the [quiz builder](http://www.theguardian.com/sport/2016/feb/19/sports-quiz-midtjylland-shaquille-oneal-shane-warne) colour scheme. Others were added by @OliverJAsh for accessibility.

I've asked @Pearson82 to update our designs and incorporate a no-js solution.